xml/System/TimeSpan.xml

```diff
@@ -128,7 +128,12 @@
  The following example instantiates a <xref:System.TimeSpan> object that represents the difference between two dates. It then displays the <xref:System.TimeSpan> object's properties.  
   
  [!code-csharp[System.TimeSpan.Class#1](~/samples/snippets/csharp/VS_Snippets_CLR_System/system.timespan.class/cs/structure1.cs#1)]
- [!code-vb[System.TimeSpan.Class#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.timespan.class/vb/structure1.vb#1)]  
+ [!code-vb[System.TimeSpan.Class#1](~/samples/snippets/visualbasic/VS_Snippets_CLR_System/system.timespan.class/vb/structure1.vb#1)]
+<<<<<<< HEAD
+ [!code-powershell[System.Timespan.Class#1](~/samples/snippets/powershell/VS_Snippets_CLR_System/system.timespan.class/ps/structure1.ps1]  
+=======
+ [!code-powershell[System.Timespan.Class#1](~/samples/snippetspowershell/VS_Snippets_CLR_System/system.timespan.class/ps/structure1.ps1]
+>>>>>>> 82e294c5709be80c8c994ce2bb7d90cc02f6860d
```

Comment:
@doctordns, this is a merge conflict that GitHub wanted you to resolve; it's marked off the two variants of the text. To resolve it, you determine which version of the text is the correct one (in this case, the first is). Then delete `<<<<<<< HEAD `, and delete the older variant as well as the GitHub tags:
```
======= 
[!code-powershell[System.Timespan.Class#1](~/samples/snippetspowershell/VS_Snippets_CLR_System/system.timespan.class/ps/structure1.ps1] 
>>>>>>> 82e294c5709be80c8c994ce2bb7d90cc02f6860d 
```